addons/rating/models/rating_mixin.py

```diff
@@ -20,6 +20,15 @@ class RatingParentMixin(models.AbstractModel):
         "Rating Satisfaction",
         compute="_compute_rating_percentage_satisfaction", compute_sudo=True,
         store=False, help="Percentage of happy ratings")
+    rating_count = fields.Integer(
+        "Ratings Count",
+        compute="_compute_rating_count",
+        help="count of ratings for visibility of stat button")
+
+    @api.depends('rating_ids')
+    def _compute_rating_count(self):
+        for record in self:
+            record.rating_count = len(self.rating_ids)
```

Comment:
Be careful with this in regard to performance, this requires fetching all of the ratings to count them in python, which might not scale well.

It is probably better to use `search_count` or even `read_group`.

See example in `_compute_rating_stats` in `rating.mixin` itself.

Note that you can probably incorporate the compute into `_compute_rating_percentage_satisfaction` and make a single query for both the count and the percentage (the percentage also uses the count internally, just have to save it into the field).